gammapy/data/event_list.py

```diff
@@ -653,6 +653,47 @@ def plot_time_map(self, ax=None):
 
         return ax
 
+    def plot_theta2_distribution(self, ax=None, number_bins=50):
+        """Plot the theta2 distribution of the events.
+        A pointing direction can be given in the same units as the event list as (ra,dec).
+        If None is given, the average of incoming directions of the events is taken.
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes` or None
+            Axes
+        number_bins: integer
+        pointing_radec: Tuple of floats or None
+
+        Returns
+        -------
+        ax : `~matplotlib.axes.Axes`
+            Axes
+
+        """
+        import matplotlib.pyplot as plt
+
+        ax = plt.gca() if ax is None else ax
+
+        events_ra = self.table[:]['RA']
+        events_dec = self.table[:]['DEC']
+
+        pointing_ra = self.pointing_radec.ra.value
+        pointing_dec = self.pointing_radec.dec.value
+
+        theta2 = (events_ra - pointing_ra)**2 + (events_dec - pointing_dec)**2
+
+        count_theta2, x_edges, y_edges = ax.hist(theta2, bins=number_bins)
```

Comment:
How about taking `**kwargs` as argument in this function, and passing it on to `ax.hist`?
Then the caller has full control over binning and other hist plotting options.